packages/vue/src/components/molecules/SfCard/SfCard.stories.js

```diff
@@ -0,0 +1,46 @@
+import { storiesOf } from "@storybook/vue";
```

Comment:
please rewrite this story to match the new standard.

Reply:
done :)